services/streammanager/kinesis/kinesismanager.go

```diff
@@ -0,0 +1,72 @@
+package kinesis
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/kinesis"
+	"github.com/tidwall/gjson"
+)
+
+// Produce creates a producer and send data to Kinesis
+func Produce(jsonData json.RawMessage) (int, string, string) {
+
+	//config := jsonData.
+	parsedJSON := gjson.ParseBytes(jsonData)
+	if parsedJSON.Get("output").Exists() {
+		parsedJSON = parsedJSON.Get("output")
+	}
+	config := parsedJSON.Get("config")
```

Comment:
Instead of typecasting each of the fields inside config, can we create a config struct that's expected for Kinesis and marshal the config directly to the struct?

Reply:
incorporated.